### ISSUE TITLE
Caravan is suddenly trapped between trees

### ISSUE THREAD

**Issue by BileDemon** [the3dfxdude/7kaa#266]
While playing the game I noticed that one of my caravans was suddenly stuck in a small forest. I don't know how it ended up like that, but I know that the path through the forest was correct. So my best guess is that while the caravan was walking through this wooded area, more trees grew around it, trapping it completely.

Unfortunately it was impossible to rescue the caravan, there was no space left to move it anywhere. I had to put the poor animal down. RIP.

Maybe there is no way or no need to fix this, but I still wanted to open it for discussion. What if this happens to other units, even the king himself? Is it possible to chop the trees down?

![7kaa3](https://user-images.githubusercontent.com/94478594/223376694-7798a258-1d27-4c0f-96fa-9c5390c65c11.jpg)

**Comment by the3dfxdude** [comment]
It's a fair complaint that it happens, but I'd rather not have open discussions on the github issue tracker. Please feel free to open a topic on the 7kfans forum if you want to discuss.

To answer your questions, I suppose this can happen in a dense wooded region! I have not had this happen to me personally. Seeing that no one has mentioned it about happening to soldier units (caravans are the most likely I think) shows this is probably really hard to do, and a rare inconvenience that probably doesn't matter. Wouldn't this have to assume that both the forward and backwards path get blocked simultaneously? Maybe it's a pathfinding issue (see #249), actually.

As far as getting something out, yes, you can burn down the trees. So I think this will fall under an enhancement request if you want to make this easier, which might happen because there are other reasons to take down trees.

If you have the save file, I can check closely. But I might not take action on this directly and just catalog it as a general, take down a tree complaint, but you can burn down the tree so there is actually a way to handle it.

**Comment by BileDemon** [comment]
I doubt it was due to a pathfinding issue. I tried to get the caravan to just move anywhere, but it couldn't even move one single tile, there were trees everywhere. So yes, I guess the path forward and the path backwards must have been blocked by trees coincidentally.

I agree that this is a really rare circumstance and losing a caravan is no big deal either. So this is obviously not a priority issue. I just wanted to report that it is a thing that happens.

When you say it it possible to burn down trees, you mean by using spitfires or something like that? I tried to get any of my units to attack trees, but it doesn't work. How do I do that?
